js/initcontext.go

```diff
@@ -105,13 +106,13 @@ func newBoundInitContext(base *InitContext, vuImpl *moduleVUImpl) *InitContext {
 
 // Require is called when a module/file needs to be loaded by a script
 func (i *InitContext) Require(arg string) (export goja.Value) {
-	var ok bool
-	if export, ok = i.exportsCache[arg]; ok {
-		return export
-	}
-	defer func() { i.exportsCache[arg] = export }()
 	switch {
 	case arg == "k6", strings.HasPrefix(arg, "k6/"):
+		var ok bool
+		if export, ok = i.k6ModulesCache[arg]; ok {
+			return export
+		}
+		defer func() { i.k6ModulesCache[arg] = export }()
```

Comment:
I need clarification on why this is required, does it expected to be _not nil_ in some cases?

Reply:
Nop, fine for me then